packages/icons/preview/import_statements.jsx

```diff
@@ -0,0 +1,16 @@
+import upperFirst from 'lodash/upperFirst';
+import camelCase from 'lodash/camelCase';
+
+const getIconName = (fileName) => fileName.substr(0, fileName.lastIndexOf('.'));
+
+export const importStatementForReactComponent = (size, fileName) => {
+  const name = getIconName(fileName);
+  const alias = upperFirst(camelCase(`${name}_icon`));
```

Comment:
I think the name is wrong, needs to be `WhateverIcon` not `Whatever_icon`.

Reply:
result is the same: 
'whatever_icon` -> 'whateverIcon' -> 'WhateverIcon'

Reply:
sorry my bad, didn't notice you pass that in

Reply:
Yes but it's a good sign that it's somehow confusing. Will add the 'Icon' part after upperFirst and camelCase